templates/_includes/overview-footer.html

```diff
@@ -7,7 +7,7 @@ <h2>Get in the Loop.</h2>
         <a href="https://github.com/rackerlabs/design-system"><img src="assets/images/GitHub-Mark-120px-plus.png" alt="GitHub"/></a>
       </div>
       <div class="hxCol-2">
-        <a href="#"><img src="assets/images/InVision-icon-black.png" alt="Invision"/></a>
+        <a href="https://rackspace.invisionapp.com/d/main#/projects"><img src="assets/images/InVision-icon-black.png" alt="Invision"/></a>
```

Comment:
note: this link requires sign-in

Reply:
Yep! I don't think we have an icon for that, unfortunately...